ide/projectapi/apichanges.xml

```diff
@@ -83,6 +83,21 @@ is the proper place.
     <!-- ACTUAL CHANGES BEGIN HERE: -->
 
     <changes>
+        <change id="project-priming-action">
+            <api name="general"/>
+            <summary>Added <code>ParentProjectProvider</code> and <code>RootProjectProvider</code> to improve project hierarchy discovery</summary>
```

Comment:
Summary isn't correct.

Reply:
Stupid copy/paste error, sorry :(
Fixed in 9fb5597448